lib/defaults/defaults.go

```diff
@@ -1118,6 +1122,13 @@ var (
 	// agents to form a cluster before commencing the operation
 	AgentWaitTimeout = 5 * time.Minute
 
+	// GenerateDebugReportTimeout specifies the maximum amount of time to wait for
+	// cluster-wide debug report generation.
+	// TODO(dmitri): this is trickier to handle correctly - ideally, this should much shorter
+	// and for the initial data wait (until there's data coming on the conn).
+	// There should be no timeout for the actual report receive operation.
+	GenerateDebugReportTimeout = 10 * time.Minute
```

Comment:
Hm, does this apply to reports generated with `gravity report` CLI command as well? If so, I think this might basically render it broken because in our support cases report generation takes a very long time, might be longer than 10 minutes for sure.

Reply:
No, it's used in `gravity install` for automatically generated crash reports atm. There's no good solution so far (I believe as far as go1.13+) as 'there's no way to have server respond with a timeout for sends without the global write timeout (which would be wrong for downloads).